website/src/components/Buttons/Skip.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+  Button,
+  ButtonProps,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  useDisclosure,
+} from "@chakra-ui/react";
+import { useTranslation } from "next-i18next";
+
+interface SkipButtonProps extends ButtonProps {
+  onSkip: () => void;
+  confirmSkip: boolean;
+}
+
+export const SkipButton = ({ ...props }: SkipButtonProps) => {
+  const { t } = useTranslation();
+  const { isOpen, onOpen: showModal, onClose: closeModal } = useDisclosure();
+
+  const onClick = () => {
+    if (props.confirmSkip) {
+      showModal();
```

Comment:
Also can you only show the modal when the input is not empty?

Reply:
I'm already doing that... that's what the confirmSkip flag is for. I'm setting that to true if taskstatus.replyValidity === "VALID" (see TaskControls.tsx), which corresponds to a non-empty reply field.

Reply:
ah thanks